packages/run-it/src/components/ShowResponse/responseUtils.tsx

```diff
@@ -73,19 +73,73 @@ export const isColumnar = (data: any[]) => {
   return !complex
 }
 
+type ItemType = 'a' | 'o' | 'v' | 'u'
```

Comment:
Be nice to have a comment describing what these are
a = array
o = object
v = value
u = undefined

Consider p for primitive instead of v for value? All of them (except maybe u) could be considered values. I guess date is not primitive.

Or define an enum

Reply:
Ok. It was internal only but I suppose it's better for someone else to understand it. I'll look at enum

Reply:
Rather than having someone obsess about "primitive" vs. "scalar" vs. "value" etc I think I'll call it a "simple" type vs. a complex/compound type (object or array).